docs/pfb.md

```diff
@@ -0,0 +1,38 @@
+# PFB
+
+The Portable Format for Biomedical Data (PFB) file format is developed by the Center for Translational Data Science at the University of Chicago (UChicago). UChicago developed [pyPFB](https://github.com/uc-cdis/pypfb), a python SDK to create, explore and modify PFB files. They have documentation on the PFB format [here](https://github.com/uc-cdis/pypfb/blob/master/docs/index.md#introduction).
+
+### TLDR;
+PFB files are Avro files with a particular required schema designed for biomedical data.
+
+
+## Building Blocks
+
+### Avro
+PFB format is built on [Avro](https://avro.apache.org/docs/), a data serilziation system. When the avro file is deserizlied, you'll find the data's schema alongside the row data, all in one compact file. 
```

Comment:
"deserialized"

Reply:
😱 Wow, I really needed spell check. Thank you!